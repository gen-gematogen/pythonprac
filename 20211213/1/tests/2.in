[22, 58, 95, 33, 47, 11, 76, 38, 0, 84, 40, 35, -5, 28, 13, 100]
