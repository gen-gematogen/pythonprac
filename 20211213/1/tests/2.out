[-5, 0, 11, 13, 22, 28, 33, 35, 38, 40, 47, 58, 76, 84, 95, 100]
